.github/workflows/test.yml

```
@@ -9,7 +9,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        python-version: [2.7, 3.6]
+        python-version: [3.6]
```

While we're here, consider updating the test matrix for the newer versions.
```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9]
```

This may require further work on clearing up version specificity in the test suite, so that's something to consider. Probably out of scope for this change.